src/zmap.1

```diff
@@ -154,6 +164,21 @@ Comma\-separated list of fields to output
 \fB\-\-output\-filter\fR
 Specify an output filter over the fields defined by the probe module\. See the output filter section for more details\.
 .
+.TP
```

Comment:
Seems that most of these changes are caused by re-running the `ronn` tool. From what I can tell, the `ronn` CLI tool takes in the `zmap.1.ronn` file and generates this `zmap.1` file in the correct format for manual pages.

Since I only changed a small typo and re-generated `zmap.1`, most of these changes must have come in `4.0.0`